From: Josh Boyer <jwboyer@fedoraproject.org>
Date: Wed, 11 Feb 2015 10:30:58 -0500
Subject: [PATCH] perf tools: Define _GNU_SOURCE on pthread_attr_setaffinity_np
 feature check

The man page for pthread_attr_set_affinity_np states that _GNU_SOURCE must
be defined before pthread.h is included in order to get the proper function
declaration.  Define this in the Makefile.

Signed-off-by: Josh Boyer <jwboyer@fedoraproject.org>
---
 tools/perf/config/feature-checks/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/perf/config/feature-checks/Makefile b/tools/perf/config/feature-checks/Makefile
index 42ac05aaf8ac..b32ff3372514 100644
--- a/tools/perf/config/feature-checks/Makefile
+++ b/tools/perf/config/feature-checks/Makefile
@@ -49,7 +49,7 @@ test-hello.bin:
 	$(BUILD)
 
 test-pthread-attr-setaffinity-np.bin:
-	$(BUILD) -Werror -lpthread
+	$(BUILD) -D_GNU_SOURCE -Werror -lpthread
 
 test-stackprotector-all.bin:
 	$(BUILD) -Werror -fstack-protector-all
-- 
2.1.0

